### PR TITLE
Added ability to change LVM Model

### PR DESCRIPTION
**Pull request by okhleif-10**
## Description

This PR adds the ability to override the Dockerfile entrypoint in the compose yaml so that a user can change the LVM model with an environment variable

## Issues

Previous mention that users could not change the LVM model

## Type of change

List the type of change like below. Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would break existing design and interface)
- [ ] Others (enhancement, documentation, validation, etc.)

## Dependencies

N/A

## Tests

Build compose.yaml and ran docker -ps --no-trunc to verify extended command

### CODE REVIEW

File: MultimodalQnA/docker_compose/intel/cpu/xeon/set_env.sh [hunk 1]
```diff
@@ -3,9 +3,10 @@
 # Copyright (C) 2024 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
-export no_proxy=${your_no_proxy}
-export http_proxy=${your_http_proxy}
-export https_proxy=${your_http_proxy}
+export no_proxy=${no_proxy}
```
Comment (mhbuehler) [inline MultimodalQnA/docker_compose/intel/cpu/xeon/set_env.sh]: Did you mean to change these? I think they have the `your_` prefix in the whole rest of the repo.

**Review by mhbuehler — COMMENTED** [review]
Did you test this? Can you describe your tests? Are the acceptable models listed in the README and did you try them all? Does the Gaudi `set_env.sh` need an analogous change?

**Comment by mhbuehler** [comment]
I've disabled the app that applied the security checks, but I guess it doesn't delete the checks that previously ran. Sorry about that.

File: MultimodalQnA/docker_compose/intel/cpu/xeon/set_env.sh (hunk 1 above)
Comment (okhleif-10) [reply MultimodalQnA/docker_compose/intel/cpu/xeon/set_env.sh]: I did for my own use, did not necessarily mean to contribute although I think it makes way more sense this way

File: MultimodalQnA/docker_compose/intel/cpu/xeon/set_env.sh [hunk 2]
```diff
@@ -3,9 +3,10 @@
...
+export http_proxy=${http_proxy}
+export https_proxy=${http_proxy}
+export HUGGINGFACEHUB_API_TOKEN=${hf_token}
```
Comment (dmsuehir) [inline MultimodalQnA/docker_compose/intel/cpu/xeon/set_env.sh]: Why do we need the HF token here?

**Review by dmsuehir — CHANGES_REQUESTED** [review]

File: MultimodalQnA/docker_compose/intel/cpu/xeon/set_env.sh [hunk 3]
```diff
@@ -15,6 +16,7 @@ export INDEX_NAME="mm-rag-redis"
 export LLAVA_SERVER_PORT=8399
 export LVM_ENDPOINT="http://${host_ip}:8399"
 export EMBEDDING_MODEL_ID="BridgeTower/bridgetower-large-itm-mlm-itc"
+export LVM_MODEL_ID="llava-hf/llava-v1.6-vicuna-13b-hf"
```
Comment (dmsuehir) [inline MultimodalQnA/docker_compose/intel/cpu/xeon/set_env.sh:18]: We should also add this to the [README](https://github.com/opea-project/GenAIExamples/blob/main/MultimodalQnA/docker_compose/intel/cpu/xeon/README.md#setup-environment-variables)

**Review by dmsuehir — CHANGES_REQUESTED** [review]
Did you see this arg for the embedding model? https://github.com/opea-project/GenAIComps/blob/main/comps/embeddings/multimodal/bridgetower/bridgetower_server.py#L109
This script is run from the [bridgetown dockerfile](https://github.com/opea-project/GenAIComps/blob/main/comps/embeddings/multimodal/bridgetower/Dockerfile#L25) similar that llava script.

**Review by dmsuehir — APPROVED** [review]
LGTM

**Pull request merged**